libs/core/concepts/include/hpx/concepts/has_member_xxx.hpp

```diff
@@ -12,16 +12,6 @@
 
 #include <type_traits>
 
-#if defined(HPX_HAVE_CXX17_VARIABLE_TEMPLATES)
-#define HPX_HAS_MEMBER_XXX_TRAIT_DEF_CXX17(MEMBER)                             \
-    template <typename T>                                                      \
-    constexpr bool HPX_PP_CAT(HPX_PP_CAT(has_, MEMBER), _v) =                  \
-        HPX_PP_CAT(has_, MEMBER)::value;                                       \
-    /**/
-#else
-#define HPX_HAS_MEMBER_XXX_TRAIT_DEF_CXX17(MEMBER) /**/
-#endif
```

Comment:
Just curious about the history of this: were varable templates at some point not going to make it into C++14?

Reply:
That's exactly the question I asked myself :/ I must have been asleep while adding this option.